src/main/java/org/skife/jdbi/v2/ClasspathStatementLocator.java

```diff
@@ -51,6 +51,7 @@ public static boolean looksLikeSql(String sql)
                || local.startsWith("create ")
                || local.startsWith("alter ")
                || local.startsWith("merge ")
+               || local.startsWith("replace ")
```

Comment:
MySQL `replace` statements were sneaking past this check. Not the underlying cause of the bug but adding this check would have prevented us from hitting it. Moved this change into its own commit